tests/controller/insertcontent.js

```diff
@@ -610,6 +612,18 @@ describe( 'DataController', () => {
 				insertHelper( '<disallowedWidget>xxx</disallowedWidget>' );
 				expect( getData( doc ) ).to.equal( '<paragraph>f[]oo</paragraph>' );
 			} );
+
+			it( 'filters out disallowed attributes', () => {
```

Comment:
I added one more test :)
If you have an idea for more tests (I'm sure you have)  I'm open :)